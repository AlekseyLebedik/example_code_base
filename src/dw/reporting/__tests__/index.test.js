import React from 'react';
import { shallow } from 'enzyme';

import Reporting from '../index';

it('Reporting unit rendered', () => {
  const wrapper = shallow(<Reporting />);
  expect(wrapper).toMatchSnapshot();
});
