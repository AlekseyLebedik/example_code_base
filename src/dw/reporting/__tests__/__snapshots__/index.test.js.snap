// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reporting unit rendered 1`] = `
<Provider
  store={
    Object {
      "@@observable": [Function],
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <Connect(Replica) />
  <Connect(Replica) />
  <Connect(Replica) />
  <Connect(Replica) />
  <withRouter(Connect(App)) />
</Provider>
`;
